rfcs/0005-fork-selection.md

```diff
@@ -0,0 +1,166 @@
+# Fork Selection
+ This RFC defines Solana's *Nakomoto Fork Selection* algorithm based on time locks. It satisfies the following properties:
+
+
+* A voter can eventually recover from voting on a fork that doesn't become the fork with the largest finality.
+* If the voters share a common ancestor then they will converge to a fork containing that ancestor no matter how they are partitioned, although it may not be the latest possible ancestor at the start of the partition.
```

Comment:
Everything after "partitioned" can be a separate sentence. And that sentence can use some word-smithing. Is there a more precise way to describe "latest possible ancestor"? And that last word, "partition", do you mean "fork"?